src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  --links-color: #00acdc;
```

Comment:
mentor meant to move it to :root selector
https://developer.mozilla.org/en-US/docs/Web/CSS/:root
it's good practice to declare variables there